cocotb/share/lib/embed/gpi_embed.c

```diff
@@ -171,6 +171,31 @@ void embed_init_python(void)
     FEXIT;
 }
 
+/**
+ * @name    Simulator cleanup
+ * @brief   Called by the simulator on shutdown.
+ * @ingroup python_c_api
+ *
+ * GILState before calling: Not held
+ *
+ * GILState after calling: Not held
+ *
+ * Makes one call to PyGILState_Ensure and one call to Py_Finalize.
+ *
+ * Cleans up reference counts for Python objects and calls Py_Finalize function.
+ */
+void embed_sim_cleanup(void)
+{
+    to_python();
+    PyGILState_Ensure();    // Don't save state as we are calling Py_Finalize
```

Comment:
Unused local variable was causing errors on linux builds, so I removed it. We shouldn't need it since `Py_Finalize()` should clean up the interpreter.